extensions/widgets/iconpicker/iconpicker.lcb

```diff
@@ -159,6 +159,7 @@ public handler OnCreate()
 	put 0 into mViewTopPosition
 	put true into mFrameBorder
 
+	put true into mShowSelectedElement
```

Comment:
What was the regression exactly? Did the default setting to true of mShowSelectedElement get removed? If so what patch did that?

Reply:
@runrevmark This patch https://github.com/livecode/livecode/pull/6534/files (which depends on https://github.com/livecode/livecode-ide/pull/1975/files) added the filtering feature, and introduced the `mShowSelectedElement` variable. But it did not initialise it.